lib/ansible/modules/cloud/digital_ocean/digital_ocean_droplet.py

```diff
@@ -0,0 +1,308 @@
+    type: bool
+  size:
+    description:
+     - This is the slug of the size you would like the droplet created with.
+    aliases: ['size_id']
+  image:
+    description:
+     - This is the slug of the image you would like the droplet created with.
+    aliases: ['image_id']
+  region:
+    description:
+     - This is the slug of the region you would like your server to be created in.
+    aliases: ['region_id']
+  ssh_keys:
+    description:
+     - array of SSH key (numeric) ID that you would like to be added to the server.
+    required: False
+  private_networking:
+    description:
+     - add an additional, private network interface to droplet for inter-droplet communication.
+    default: False
+    type: bool
+  user_data:
+    description:
+      - opaque blob of data which is made available to the droplet
+    required: False
+  ipv6:
+    description:
+      - enable IPv6 for your droplet.
+    required: False
+    default: False
+    type: bool
+  wait:
+    description:
+     - Wait for the droplet to be active before returning.  If wait is "no" an ip_address may not be returned.
+    required: False
+    default: True
+    type: bool
+  wait_timeout:
+    description:
+     - How long before wait gives up, in seconds, when creating a droplet.
+    default: 120
+  backups:
+    description:
+     - indicates whether automated backups should be enabled.
+    required: False
+    default: False
+    type: bool
+  monitoring:
+    description:
+     - indicates whether to install the DigitalOcean agent for monitoring.
+    required: False
+    default: False
+    type: bool
+  tags:
+    description:
+     - List, A list of tag names as strings to apply to the Droplet after it is created. Tag names can either be existing or new tags.
+    required: False
+  volumes:
+    description:
+     - List, A list including the unique string identifier for each Block Storage volume to be attached to the Droplet.
+    required: False
+  oauth_token:
+    description:
+     - DigitalOcean OAuth token. Can be specified in C(DO_API_KEY), C(DO_API_TOKEN), or C(DO_OAUTH_TOKEN) environment variables
+    aliases: ['API_TOKEN']
+    required: True
+requirements:
+  - "python >= 2.6"
+'''
+
+
+EXAMPLES = '''
+- name: create a new droplet
+  digital_ocean_droplet:
+    state: present
+    name: mydroplet
+    oauth_token: XXX
+    size: 2gb
+    region: sfo1
+    image: ubuntu-16-04-x64
+    wait_timeout: 500
+  register: my_droplet
+
+- debug:
+    msg: "ID is {{ my_droplet.droplet.id }}"
+
+- name: ensure a droplet is present
+  digital_ocean_droplet:
+    state: present
+    command: droplet
+    id: 123
+    name: mydroplet
+    oauth_token: XXX
+    size: 2gb
+    region: sfo1
+    image: ubuntu-16-04-x64
+    wait_timeout: 500
+'''
+
+
+RETURN = '''
+# Digital Ocean API info https://developers.digitalocean.com/documentation/v2/#droplets
+data:
+    description: a DigitalOcean Droplet
+    returned: changed
+    type: dict
+    sample: {
```

Comment:
Please also update the response document to represent the ip addresses:

```
        "ip_address": "104.248.118.172",
        "ipv6_address": "2604:a880:400:d1::90a:6001",
        "private_ipv4_address": "10.136.122.141",
        "droplet": {....}
```

Reply:
Looks like IP address information has been added